src/lexical_grammar.rs

```diff
@@ -82,12 +100,18 @@ pub struct PrivateIdentifier {
     pub payload: IdentifierName
 }
 
-#[derive(Debug, FromPest)]
+#[derive(Debug, FromPest, Eq, PartialEq)]
```

Comment:
```suggestion
#[derive(Debug, Eq, FromPest, PartialEq)]
```

We should keep an alphabet order here.